docs/src/operate/c8y/connect.md

```diff
@@ -58,183 +59,236 @@ in this case we are told to run `tedge config set c8y.url <value>`.
 
-To create the bridge use the [`tedge connect`](../../references/cli/tedge-connect.md) command.
+#### Registering a device using the Device Management UI {#cumulocity-ca-register-ui}
 
-```sh
-sudo tedge connect c8y
-```
-
-```text title="Output"
-Checking if systemd is available.
-
-Checking if configuration for requested bridge already exists.
+1. Open the Cumulocity *Device Management* UI Application
 
-Validating the bridge certificates.
+1. Navigate to *Devices* &rarr; *Registration*
 
-Creating the device in Cumulocity cloud.
+1. Click *Register device* and select *General*
 
-Saving configuration for requested bridge.
+    <p align="center">
+        <img width="60%" src={require('./images/register-c8y-ca-register-1-device.png').default} alt="Register device" />
+    </p>
 
-Restarting mosquitto service.
+1. In the *Register general devices* dialog box, select the "Create device certificates during device registration" option
 
-Awaiting mosquitto to start. This may take up to 5 seconds.
+1. Fill in the following required fields and click *Next*
 
-Enabling mosquitto service on reboots.
+    <p align="center">
+        <img width="60%" src={require('./images/register-c8y-ca-register-2-form-details.png').default} alt="Register device form" />
+    </p>
 
-Successfully created bridge connection!
+    |Field|Description|
+    |----|-----------|
+    |Device ID|Device's which will be used as the Common Name in the device's certificate|
+    |One-time password|One-time password that the device can use to download its device certificate. Maximum length of 32 characters|
 
-Sending packets to check connection. This may take up to 2 seconds.
+    :::tip
+    Make sure you copy the one-time password as this will be required on the device.
+    :::
 
-Connection check is successful.
+1. Close the form
 
-Checking if tedge-mapper is installed.
+    <p align="center">
+        <img width="60%" src={require('./images/register-c8y-ca-register-3-form-complete.png').default} alt="Close registration form" />
+    </p>
 
-Starting tedge-mapper-c8y service.
+1. On the device, run the following command to download the device's certificate (you will be prompted for the one-time password)
 
-Persisting tedge-mapper-c8y on reboot.
+    <UserContext>
 
-tedge-mapper-c8y service successfully started and enabled!
+    ```sh
+    sudo tedge cert download c8y --device-id "$DEVICE_ID"
 
-Enabling software management.
+    # Or provide the one-time password via the flag
+    sudo tedge cert download c8y \
+        --device-id "$DEVICE_ID" \
+        --one-time-password "$DEVICE_ONE_TIME_PASSWORD"
+    ```
 
-Checking if tedge-agent is installed.
+    </UserContext>
 
-Starting tedge-agent service.
+#### Register a device using go-c8y-cli command {#cumulocity-ca-register-cli}
 
-Persisting tedge-agent on reboot.
+Below describes the steps to register a new device using a combination of [go-c8y-cli](https://goc8ycli.netlify.app/) commands and %%te%%.
```

Comment:
I would stress that this is not a second step after "Registering a device using the Device Management UI" but an alternative option.

Reply:
Pushed some changes that hopefully resolve it 547315910d5746344b3303f217a0f41e86e89ff3